tools/experiments/experiments.js

```diff
@@ -348,7 +386,7 @@ function getAmpConfig() {
   xhr.send(null);
   return promise
     .then(text => {
-      const match = text.match(/self\.AMP_CONFIG=([^;]+)/);
+      const match = text.match(/self\.AMP_CONFIG=(\{.+?\})/);
```

Comment:
What exactly is this trying to do that it's regex-matching JS? Not that it should be fixed in this PR, but curious

Reply:
Read the `AMP_CONFIG`, which is a JavaScript object (which happens to be JSON-compatible), so that it can see which experiments you're opted-in to